File: 백준/Bronze/2750. 수 정렬하기/수 정렬하기.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int N = sc.nextInt();
        int[] numArr = new int[N];
        
        for(int i = 0; i < N; i++){
            numArr[i] = sc.nextInt();
        }
        
        Arrays.sort(numArr);
        
        for(int i = 0; i < N; i++) {
            System.out.println(numArr[i]);
        }
    }
}

File: 백준/Bronze/2750. 수 정렬하기/수 정렬하기.cc
#include <iostream>
#include <algorithm>
using namespace std;

int main() {
	int n;
	cin >> n;

	int* numArr = new int[n];

	for (int i = 0; i < n; i++)
	{
		cin >> numArr[i];
	}

	sort(numArr, numArr + n);

	for (int i = 0; i < n; i++)
	{
		cout << numArr[i] << endl;
	}
}